src/plugins/vis_types/xy/public/utils/accessors.tsx

```diff
@@ -9,7 +9,12 @@
 import { AccessorFn, Accessor } from '@elastic/charts';
 import { BUCKET_TYPES } from '../../../../data/public';
 import { FakeParams } from '../../../../visualizations/public';
-import { Aspect } from '../types';
+import type { Aspect } from '../types';
+
+interface Dimension {
```

Comment:
This type here confuses me because we already have defined a type `Dimension` in the types folder. Is it different from this type? If yes, it should have a different name. If no, the already existing type should be used